src/main/resources/META-INF/plugin.xml

```diff
@@ -430,7 +431,35 @@
                 implementation="com.redhat.devtools.lsp4ij.features.semanticTokens.LSPSemanticTokensHighlightVisitor"/>
         <colorSettingsPage
                 implementation="com.redhat.devtools.lsp4ij.features.semanticTokens.SemanticTokensColorSettingsPage"/>
+    </extensions>
 
+    <!-- Semantic tokens-based file view provider -->
+    <extensions defaultExtensionNs="com.intellij">
```

Comment:
Split `extensions` here so that we can have the optional dependencies for Community Edition IDEs group with the other file view provider EP registrations.